fvm/evm/stdlib/contract_test.go

```diff
@@ -3428,7 +3436,7 @@ func TestEVMCreateCadenceOwnedAccount(t *testing.T) {
 	require.NoError(t, err)
 
 	expected := cadence.NewArray([]cadence.Value{
-		cadence.UInt8(4), cadence.UInt8(0),
+		cadence.UInt8(5), cadence.UInt8(0),
```

Comment:
these changes are due to an extra uuid allocation for the heartbeat resource on EVM deployment